packages/wp-story-editor/src/components/checklist/priority/index.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import {
+  useConfig,
+  PublisherLogoSize,
+  StoryMissingExcerpt,
+  StoryMissingTitle,
+  StoryPosterSize,
+  StoryPosterAttached,
+  StoryTitleLength,
+  VideoElementMissingPoster,
+  VideoOptimization,
+  StoryAmpValidationErrors,
+  PublisherLogoMissing,
+  useFFmpeg,
+} from '@web-stories-wp/story-editor';
+
+/**
+ * Internal dependencies
+ */
+import StoryMissingPublisherName from './storyMissingPublisherName';
+
+function Priority() {
+  const { canManageSettings, hasUploadMediaAction } = useConfig(
+    ({ capabilities }) => ({
+      canManageSettings: capabilities.canManageSettings,
+      hasUploadMediaAction: capabilities.hasUploadMediaAction,
+    })
+  );
+  const { isTranscodingEnabled } = useFFmpeg();
+
+  return (
+    <>
+      <StoryMissingTitle />
+      {canManageSettings && <StoryMissingPublisherName />}
+      <StoryTitleLength />
+      <StoryMissingExcerpt />
+      {hasUploadMediaAction && <StoryPosterAttached />}
+      {hasUploadMediaAction && <StoryPosterSize />}
+      {hasUploadMediaAction && <PublisherLogoMissing />}
+      {hasUploadMediaAction && <PublisherLogoSize />}
+      {hasUploadMediaAction && <VideoElementMissingPoster />}
+      {/* `isTranscodingEnabled` already checks for `hasUploadMediaAction` */}
+      {isTranscodingEnabled && <VideoOptimization />}
```

Comment:
If we'd move this `isTranscodingEnabled` check within `VideoOptimization` then we wouldn't have to import `useFFmpeg` here.

Though that could result in some unnecessary renders 🤔

Reply:
Yeah, there [are quite a few things happening](https://github.com/google/web-stories-wp/blob/main/packages/story-editor/src/components/checklist/checks/videoOptimization.js#L114-L195) in that component, I would want to avoid running those. But I have a middle ground, I created a wrapper component for `VideoOptimization`, and this way we would not have to export `useFFmpeg`.

Done in https://github.com/google/web-stories-wp/pull/9710/commits/8553a4356e5bb5129c57ee0cb3c6f7b161b95732